kged/cypress/integration/roomtest.js

```diff
@@ -1,3 +1,9 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+})
+
```

Comment:
@evktalo I needed this part to catch the error in last test, where the interactive dialog is tested. If you run the test using cypress "interactive" window with yarn cy:open, the last test fails 
![image](https://user-images.githubusercontent.com/36008914/73156515-b9146a00-40e6-11ea-9b8a-1ea7f666ed2c.png) After your comment I tried the tests in command line and noticed that the last test actually passes. I will remove this part, since it seems like the interaction cannot be tested in cypress interactive window.

Reply:
Okay, thanks for the explanation! Could you file an issue about the test failing in interactive mode?